Api/IriConverter.php

```diff
@@ -134,4 +143,22 @@ private function getRouteName(ResourceInterface $resource, $prefix)
             }
         }
     }
+
+    /**
+     * @param ResourceInterface $resource
+     * @param array $context
+     *
+     * @return string
+     */
+    private function getIdentifierFromResource(ResourceInterface $resource, array $context = array())
```

Comment:
The `$context` parameter seems to be never used, so the method can be drastically simplified by just removing it and all `isset()` tests.